### DISCUSSION TITLE
Problems installing SSL certificate in Weblate packaged by Bitnami for GCP

### DISCUSSION THREAD

**Discussion by CtrlShiftSergio** [WeblateOrg/weblate#10286] [Q&A]
Hello, everyone! Newbie here…

I have deployed [Weblate Packaged By Bitnami For Google Cloud Platform](https://docs.bitnami.com/google/apps/weblate/). Now I'm trying to set up an SSL certificate using Let's Encrypt and Bitnami's HTTPS configuration tool as per [Auto-configure a Let's Encrypt certificate](https://docs.bitnami.com/google/faq/administration/generate-configure-certificate-letsencrypt/).

As per the screenshot at the bottom, I keep running into the same error—it seems a service is using ports 80 and 443, so Let's Encrypt can't use them. The service appears to be "httpd," but I have no idea how to stop it, so that I could run LE and then start httpd again.

Do you have any ideas on how I could solve this? Thank you so much!

![image](https://github.com/WeblateOrg/weblate/assets/149143357/2c6dcab5-c25b-4512-911f-26ec45295d93)

**Comment by nijel** [comment]
This is really more a question to somebody knowing Bitnami stack, it's not really Weblate specific....
